tests/test_other.py

```diff
@@ -3195,6 +3195,24 @@ def test_warn_unaligned(self):
     assert 'emcc: warning: unaligned store' in output[1], output[1]
     assert '@line 11 "src.cpp"' in output[1], output[1]
 
+  def test_on_abort(self):
+    cmd = [PYTHON, EMCC, path_from_root('tests', 'hello_world.c'), '-s', 'WASM=1']
+
+    self.clear()
+    proc = Popen(cmd, stdout=PIPE, stderr=PIPE)
+    output, err = proc.communicate()
+
+    expectedOutput = "Module['onAbort'] was called"
+
+    os.remove('a.out.wasm')
+    f = open('a.out.js', 'a')
+    f.write("Module['onAbort'] = function () { console.log('%s') }" % expectedOutput)
+    f.close()
+
+    devnull = open(os.devnull, 'w')
```

Comment:
can we remove the devnull parts? it's out if stderr is logged to the console as the test runs (easier to debug, when necessary)